### PR TITLE
Change WebVR references to WebXR

### PR DESCRIPTION
**Pull request by jacob-willden**
**Description:**
Updating the references to the deprecated WebVR API on the "VR Headsets & WebVR Browser page". Since the page URL includes the term WebVR, either keeping the old naming in the page title or creating a redirect may be needed.

### CODE REVIEW

File: docs/introduction/vr-headsets-and-webxr-browsers.md [hunk 1]
```diff
@@ -145,11 +145,11 @@ front-facing changes to A-Frame developers, involving mostly renaming of APIs.
 - Chrome (WebXR under origin trials)
 - Exokit (experimental early support)
 
-[webvrpolyfill]: https://github.com/googlevr/webvr-polyfill
```
Comment (dmarcos) [inline docs/introduction/vr-headsets-and-webxr-browsers.md:148]: A-Frame still uses the WebVR polypill and no plans to migrate for now. We can revert this change

**Comment by dmarcos** [comment]
Thanks just a couple of small changes and ready to merge

**Comment by dmarcos** [comment]
Thanks for the patience. I'm merging this and I'll replace the link to the WebVR polyfill.

Congrats on your first contribution! 🥳

**Comment by jacob-willden** [comment]
Great! Thank you so much.
